.vscode/settings.json

```
@@ -18,6 +18,5 @@
   "eslint.nodePath": ".yarn/sdks",
   "prettier.prettierPath": ".yarn/sdks/prettier/index.js",
   "typescript.enablePromptUseWorkspaceTsdk": true,
-  "jest.jestCommandLine": "yarn jest",
-  "editor.bracketPairColorization.enabled": true
+  "jest.jestCommandLine": "yarn jest"
```

Can we keep `editor.bracketPairColorization.enabled` on the file
```suggestion
  "jest.jestCommandLine": "yarn jest",
  "editor.bracketPairColorization.enabled": true,
```